Cargo.toml

```diff
@@ -61,5 +61,7 @@ default = ["integration-tests"]
 
 integration-tests = []
 
+native-tls = ["reqwest/native-tls"]
```

Comment:
**nit**: Can we remove the newline between `integration-tests` and `native-tls`?